src/components/molecules/Navigation.js

```diff
@@ -1,59 +1,126 @@
 import React from "react";
+import { useRouter } from "next/router";
 import Link from "next/link";
 import styled from "styled-components";
+import { useAuth0 } from "@auth0/auth0-react";
 
 const Nav = styled("nav")`
-	width: 100%;
-	top: 0;
-	position: absolute;
+	height: 75px;
+	position: relative;
+	background-color: var(--foreground);
+	box-shadow: 0px 3px 6px rgb(0, 0, 0, 0.16);
 `;
 
-const NavList = styled("ul")`
-    display: flex;
-    justify-content: center;
-    align-items: center;
+const NavContent = styled("div")`
+	max-width: 1000px;
+	margin: 0 auto;
+	display: grid;
+	grid-template-columns: 2fr 4fr 2fr;
+
+	ul {
+		display: flex;
+		list-style: none;
+		color: var(--text);
+	}
+`;
+
+const NavItem = styled("li")`
+	margin: auto 0;
+	padding: 0 12.5px;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 500;
+	font-size: 13px;
+
+	${({ $active }) => $active && `
+		font-weight: 700;
+		border-bottom: 2px solid var(--cs-blue);
+	`}
+
+	a {
+		color: inherit;
+
+		:hover {
+			text-decoration: none;
+			color: var(--cs-blue);
+		}
+	}
 `;
 
 const Logo = styled("img")`
-	object-fit: contain;
-	cursor: pointer;
-	
 	&& {
+		padding: 12.5px;
 		height: 50px;
-		padding: 20px;
-    }
+		display: block;
+	}
+`;
+
+const PageLinks = styled("ul")`
+	display: flex;
+	justify-content: center;
+`;
+
+const ProfileLinks = styled("ul")`
+	margin-left: auto;
 `;
 
 function Navigation() {
+	const { isAuthenticated, loginWithRedirect, logout } = useAuth0();
+	const router = useRouter();
+
+	const activeTab = router.pathname.replace("/", "").split("/")[0];
+
 	return (
-		<Nav className="uk-navbar-container uk-box-shadow-large" data-uk-navbar>
-			<div className="uk-navbar-left">
-				<NavList className="uk-navbar-nav">
-					<Link href="/" passHref>
-						<Logo
-							className="uk-navbar-item uk-logo"
-							alt="CodeSupport Logo"
-							src="/logo.png"
-							draggable="false"
-						/>
-					</Link>
-					<li>
+		<Nav>
+			<NavContent>
+				<Link href="/" passHref>
+					<Logo
+						className="uk-navbar-item uk-logo"
+						alt="CodeSupport Logo"
+						src="/logo.png"
+						draggable="false"
+					/>
+				</Link>
+				<PageLinks>
+					<NavItem $active={activeTab === ""}>
 						<Link href="/" passHref>
 							<a>Home</a>
 						</Link>
-					</li>
-					<li>
+					</NavItem>
+					<NavItem $active={activeTab === "community"}>
+						<Link href="/community" passHref>
+							<a>Community</a>
+						</Link>
+					</NavItem>
+					<NavItem $active={["articles", "article"].includes(activeTab)}>
 						<Link href="/articles" passhref>
 							<a>Articles</a>
 						</Link>
-					</li>
-					<li>
+					</NavItem>
+					<NavItem $active={["resources", "resource"].includes(activeTab)}>
```

Comment:
Probably a bit of an edge case, but what about user-defined(ish) urls? Think if an article for example contains the word "resource"?

Reply:
I think that would be fine, my logic (should if I wrote it correctly) set `activeTab` to the first part of the URL, so if an article was `articles/resources-are-cool` it would set `activeTab`. Since UGC will not ever be the base of a path I think it's all good.

Reply:
Ah yep makes sense! Tested yeah and works as intended, fair enough!